projects/luajit/build.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google LLC
+# Modifications copyright (C) 2021 ISP RAS
+# Modifications copyright (C) 2023 Sergey Bronnikov
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+CC=clang
+CXX=clang++
+CFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+CXXFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+
+cd /testdir
+
+: ${LD:="${CXX}"}
+: ${LDFLAGS:="${CXXFLAGS}"}  # to make sure we link with sanitizer runtime
+
+cmake_args=(
+    -DUSE_LUAJIT=ON
+    -DLUA_VERSION=224129a8e64bfa219d35cd03055bf03952f167f6
+    -DOSS_FUZZ=OFF
+    -DENABLE_ASAN=OFF
+    -DENABLE_UBSAN=OFF
```

Comment:
Is it ok that sanitizers are turned off?

Reply:
Yes, vanilla LuaJIT doesn't support sanitizers.